playbook-website/package.json

```diff
@@ -24,6 +24,7 @@
     "react": "^17.0.2",
     "react-docgen": "^5.4.3",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^6.16.0",
```

Comment:
Added this as a preliminary thinking that the story originally covered some of this. 
We can just leave this in here for now, if that's cool. 
I know we will use them. 

Or if preferable, I can remove until we use them in a follow-up story.